### DISCUSSION TITLE
How to get sponsored download version?

### DISCUSSION THREAD

**Discussion by ktalley-cinc** [massCodeIO/massCode#490] [General]
Hello Anton,
I gave a small ($5 USD) development donation for massCode back in 2020 -- is it still possible to get the "sponsored" version, or do I need to make another (more current) donation to have access to that?   (I tried emailing you directly with PayPal screenshots at reshetov.art@gmail.com, but I got failed delivery responses)  
Thank you!
Kevin T.

**Comment by antonreshetov** [comment]
The sponsor version is always available at the link (I send this link to all sponsors)
